app/src/main/java/com/conupods/Settings/DefaultPreferences.java

```diff
@@ -0,0 +1,28 @@
+package com.conupods.Settings;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+import com.conupods.OutdoorMaps.View.MainActivity;
+import com.conupods.R;
+
+public class DefaultPreferences {
+
+    private SharedPreferences mPreferences;
+    private SharedPreferences.Editor mPrefEdit;
+
+    public DefaultPreferences(MainActivity current){
+        mPreferences = current.getApplicationContext().getSharedPreferences("Preferences",Context.MODE_PRIVATE);
+        mPrefEdit = mPreferences.edit();
+    }
```

Comment:
instead of passing the main activity, pass SharedPreference to the default pref. class. Why does DefaultPreferences need to know about MainActivity